src/cache_status_query_function.cpp

```diff
@@ -215,6 +215,65 @@ void CacheAccessInfoQueryTableFunc(ClientContext &context, TableFunctionInput &d
 	output.SetCardinality(count);
 }
 
+//===--------------------------------------------------------------------===//
+// Wrapped cache filesystem query function
+//===--------------------------------------------------------------------===//
+struct WrappedFilesystemsData : public GlobalTableFunctionState {
+	vector<string> wrapped_filesystems;
+
+	// Used to record the progress of emission.
+	uint64_t offset = 0;
+};
+
+unique_ptr<FunctionData> WrappedCacheFileSystemsFuncBind(ClientContext &context, TableFunctionBindInput &input,
+                                                         vector<LogicalType> &return_types, vector<string> &names) {
+	D_ASSERT(return_types.empty());
+	D_ASSERT(names.empty());
+
+	return_types.reserve(1);
+	names.reserve(1);
+
+	// Wrapped cache filesystem name.
+	return_types.emplace_back(LogicalType::VARCHAR);
+	names.emplace_back("wrapped_filesystems");
```

Comment:
I think the naming and returned rows are inconsistent, if you name it as "wrapped_filesystems", it should be the one being named, so HTTPFS, S3FS, etc.